screens/NewSubmission.tsx

```diff
@@ -1,19 +1,180 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useContext, useState } from 'react';
+
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { ActivityIndicator, StyleSheet, Text, TextInput, View, Alert } from 'react-native';
+import { useForm, Controller, type SubmitHandler } from 'react-hook-form';
+import { z } from 'zod';
+import { zodResolver } from "@hookform/resolvers/zod";
+
+import { instance as axiosInstance } from '../helpers/axiosConfig';
+import getErrorMessage from '../helpers/getErrorMessage';
+import { DrawerScreenProps } from '@react-navigation/drawer';
+import { DrawerParamList } from '../Root';
+import Colors from '../constants/Colors';
+
+
+const schema = z.object({
+  title: z
+    .string()
+    .min(1, { message: "Title is required" })
+    .max(255, { message: "Title is too long" }),
+  symptoms: z
+    .string() 
+    .min(1, { message: "Symptoms are required" })
+    .max(1023, { message: "Symptoms are too long" })
+});
+
+type NewSubmissionFormData = z.infer<typeof schema>;
+
+export default function NewSubmissionScreen({ navigation }: DrawerScreenProps<DrawerParamList, 'NewSubmission'>) {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
you can infer the value here, no need to specify it's a boolean

```suggestion
  const [isLoading, setIsLoading] = useState(false);
```